packages/rpc-core/src/rpc-methods/getFeeForMessage.ts

```diff
@@ -0,0 +1,18 @@
+import { Base64EncodedBytes, Commitment, RpcResponse, Slot, U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+/** Fee corresponding to the message at the specified blockhash */
+type GetFeeForMessageApiResponse = RpcResponse<U64UnsafeBeyond2Pow53Minus1 | null>;
+
+export interface GetFeeForMessageApi {
+    /**
+     * Returns the fee the network will charge for a particular Message
+     */
+    getFeeForMessage(
+        /** Base-64 encoded message */
+        message: Base64EncodedBytes,
```

Comment:
We should really create an opaque type here to represent ‘a bunch of bytes that are definitely a serialized message.’

1. Create such an opaque type
2. Update `packages/transaction/src/serializers/message.ts` to return that type from `serialize`
3. Make this function only accept that type